src/library_html5.js

```diff
@@ -362,13 +362,9 @@ var LibraryHTML5 = {
     return GL.offscreenCanvases[target.substr(1)] // Remove '#' prefix
     // If not found, if one is querying by using DOM tag name selector 'canvas', grab the first
     // OffscreenCanvas that we can find.
-     || (target == 'canvas' && Object.keys(GL.offscreenCanvases)[0])
+     || (target == 'canvas' && Object.keys(GL.offscreenCanvases) && GL.offscreenCanvases[Object.keys(GL.offScreenCanvases)[0]])
```

Comment:
This line looks like it might be more complex that it needs to be.  

About just `target == 'canvas' && Object.values(GL.offscreenCanvases)[0]`?

Can we make this a separate PR?    Ideally with a test.  I guess we have very little testing for this since the existing code seems simply broken?

Reply:
I agree with that change and will push it here, but I think this whole lookup thing is pretty hacked together/"best effort".